README.md

```diff
@@ -1222,30 +1263,33 @@ The `payment` objects include additional transactional metadata:
 
 Places an order to exchange currencies.
 
+<!-- <div class='multicode'> -->
 
+*REST*
 
-```js
+```
```

Comment:
The `js` is missing here, which makes it show up nicer on github (syntax highlighting)